### PR TITLE
Tests beta

### PR DESCRIPTION
**Pull request by A-Vzer**

### CODE REVIEW

File: test/rules/beta/test_out.jl [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        @test_rules [ with_float_conversions = true ] Beta(:out, Marginalisation) [
+            (input = (m_a = PointMass(1.0), m_b = PointMass(2.0)), output = Beta(1.0, 2.0)),
+            (input = (m_a = PointMass(2.0), m_b = PointMass(2.0)), output = Beta(2.0, 2.0)),
+            (input = (m_a = PointMass(3.0), m_b = PointMass(3.0)), output = Beta(3.0, 3.0)),
+        ]
+
+    end
+
+    @testset "Belief Propagation: (q_a::PointMass, q_b::PointMass)" begin
```
Comment (bvdmitri) [inline test/rules/beta/test_out.jl:20]: @A-Vzer  Rules that use `q_` variables in it are **Variational Message Passing** rules, not Belief Propagation. Could you please change it everywhere? If all inputs are `m_` than it is the **Belief Propagation**

File: test/rules/beta/test_marginals.jl [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module RulesBetaMarginalsTest
+
+using Test
+using ReactiveMP
+using Random
+import ReactiveMP: @test_marginalrules
+
+@testset "marginalrules:Beta" begin
+
+     @testset "m_out, m_a, m_b: (m_out::Beta, m_a::PointMass, m_b::PointMass)" begin
```
Comment (bvdmitri) [inline test/rules/beta/test_marginals.jl:10]: Can you please change title to reflect marginal name `:out_a_b`.
Something like 
```
@testset ":out_a_b (m_out::Beta, m_a::PointMass, m_b::PointMass)" begin
...
end
```

**Comment by bvdmitri** [comment]
Approved, but requires minor changes and we can merge.

**Review by bartvanerp — APPROVED** [review]
Good! After fixing naming conventions as denoted by @bvdmitri we can merge.

**Review by bvdmitri — APPROVED** [review]

**Pull request merged**